air-quality-ui/system_tests/ui/city_page.spec.ts

```diff
@@ -61,6 +57,43 @@ test('Legend deselect removes chosen site', async ({ page, cityPage }) => {
   await expect(cityPage.textFinder('Centro')).not.toBeVisible()
 })
 
+test.describe('City page resolution tests', () => {
+  ;[
+    {
+      width: 960,
+      height: 1080,
+      screenCenter: { x: 480, y: 540 },
+    },
+    {
+      width: 1366,
+      height: 768,
+      screenCenter: { x: 683, y: 384 },
+    },
+  ].forEach(({ width, height, screenCenter }) => {
+    test.use({
+      viewport: { width: width, height: height },
+    })
+    test(`Verify that the resolution (width: ${width} height: ${height}) does not affect the ability to view the charts`, async ({
+      page,
+      cityPage,
+    }) => {
+      await gotoPage(page, 'city/Rio%20de%20Janeiro')
+      await cityPage.waitForAllGraphsToBeVisible()
+
+      expect(cityPage.aqiChart).toBeInViewport()
+      expect(cityPage.no2Chart).toBeInViewport()
+      expect(cityPage.pm2_5Chart).toBeInViewport()
+      expect(cityPage.o3Chart).toBeInViewport()
+      await page.mouse.move(screenCenter.x, screenCenter.y)
+      await page.mouse.wheel(1080, 0)
```

Comment:
We can make sure that the entirety of the element is in view like so `toBeInViewport({ ratio: 1 })`